### ISSUE TITLE
Command succeeded, but not all targets were analyzed, MAC M1.

### ISSUE THREAD

**Issue by afsaredrisy** [google/mediapipe#1516]
I am trying to build new Mediapipe in MAC M1 chip. Installed bazel bazel 3.7.2 and run setup_android_sdk_and_ndk.sh and ,`build_android_examples.sh`. I check in WORKSPACE and SDK & NDK path is setup correct. BTW I follow [https://google.github.io/mediapipe/getting_started/android.html](These instructions to import).  When I am trying to sync in android studio then I am getting following error log. 
```
Syncing project: Sync (incremental)...
Updating VCS...
Running Bazel info...
Command: bazel info --tool_tag=ijwb:AndroidStudio --curses=no --color=yes --progress_in_terminal_title=no --host_crosstool_top=@bazel_tools//tools/cpp:toolchain --

Computing VCS working set...
Sync targets from project view targets:
  //mediapipe/examples/android/...:all
  //mediapipe/java/...:all

Building Bazel targets...
Command: bazel build --tool_tag=ijwb:AndroidStudio --keep_going --build_event_binary_file=/var/folders/6g/d06l6l595qzfvgb5ysyb0p9m0000gn/T/intellij-bep-ce358f3e-0450-40c4-8723-ed0012e78ae8 --nobuild_event_binary_file_path_conversion --curses=no --color=yes --progress_in_terminal_title=no --host_crosstool_top=@bazel_tools//tools/cpp:toolchain --noexperimental_run_validations --aspects=@intellij_aspect//:intellij_info_bundled.bzl%intellij_info_aspect "--override_repository=intellij_aspect=/Users/afsar/Library/Application Support/Google/AndroidStudio4.1/plugins/aswb/aspect" --output_groups=intellij-info-android,intellij-info-generic,intellij-info-java,intellij-render-resolve-android,intellij-resolve-android,intellij-resolve-java -- //mediapipe/examples/android/...:all //mediapipe/java/...:all

Loading: 
Loading: 0 packages loaded
Loading: 29 packages loaded
    currently loading: mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d
Analyzing: 82 targets (30 packages loaded, 0 targets configured)
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handdetectiongpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework/jni:class_registry': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/posetrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/components:android_camerax_helper': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/holistictrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d:manifest_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handtrackinggpu:handtrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/upperbodyposetrackinggpu:upperbodyposetrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectiongpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework/jni:mediapipe_framework_jni': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/holistictrackinggpu:holistictrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:android_framework_no_proguard': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/templatematchingcpu:templatematchingcpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectiongpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/components:android_microphone_helper': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/holistictrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:android_framework': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handtrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/templatematchingcpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectiongpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:mediapipe_exception_android': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/basic:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework/jni:mediapipe_android_framework_jni': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/iristrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facemeshgpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/faceeffect:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:android_framework_no_mff': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework/jni:register_natives': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectiongpu:objectdetectiongpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectiongpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/upperbodyposetrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking:instantmotiontracking_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/basic:helloworld': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facemeshgpu:facemeshgpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework/jni:jni_util': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectiongpu:facedetectiongpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handtrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objecttrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/basic:basic_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectioncpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/hairsegmentationgpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d:binary_graph': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facemeshgpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/iristrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/hairsegmentationgpu:hairsegmentationgpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/hairsegmentationgpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objecttrackinggpu:objecttrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectioncpu:objectdetectioncpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectioncpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objecttrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/posetrackinggpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking:instantmotiontracking': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectioncpu:facedetectioncpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handdetectiongpu:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/faceeffect:mediapipe_jni_lib': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking:sticker_buffer_java_proto_lite': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetectioncpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/faceeffect:faceeffect': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/upperbodyposetrackinggpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/components:android_components': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/glutil:glutil': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/objectdetection3d:objectdetection3d': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/basic:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/iristrackinggpu:iristrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:framework_proto_lite': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/handdetectiongpu:handdetectiongpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/facedetectioncpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/templatematchingcpu:libmediapipe_jni.so': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/examples/android/src/java/com/google/mediapipe/apps/posetrackinggpu:posetrackinggpu': it will not be built
WARNING: errors encountered while analyzing target '//mediapipe/java/com/google/mediapipe/framework:android_core': it will not be built
INFO: Analysis succeeded for only 8 of 82 top-level targets
INFO: Analyzed 82 targets (50 packages loaded, 101 targets configured).

INFO: Found 8 targets...
[0 / 1] [Prepa] BazelWorkspaceStatusAction stable-status.txt
ERROR: command succeeded, but not all targets were analyzed
INFO: Elapsed time: 4.113s, Critical Path: 0.08s
INFO: 1 process: 1 internal.
FAILED: Build did NOT complete successfully
FAILED: Build did NOT complete successfully
INFO: Build Event Protocol files produced successfully.
FAILED: Build did NOT complete successfully
build invocation result: build_error
Sync was successful, but there were Bazel build errors. The project may not be fully updated or resolve until fixed. If the errors are from your working set, please uncheck 'Bazel > Sync > Expand Sync to Working Set' and try again.
Parsing build outputs...
Total rules: 0, new/changed: 0, removed: 0
Reading IDE info result...
Reading IDE info result...
Updating target map
Loaded 0 aspect files, total size 0kB
Target map size: 0
Prefetching output artifacts...
Reading jdeps files...
Loaded 0 jdeps files, total size 0kB
Reading package manifests...
Java content entry count: 1
Filtering empty jars...
Fetching JARs to track empty status..
Filtered 0 JARs, in 0ms
Updating Unpacked AAR libraries...
Fetching aar files...
Updating Jar Cache...
Prefetching files...
Refreshing files
Computing directory structure...
Initializing project SDKs...
Committing project structure...
Workspace has 0 libraries
Android resource module count: 0, order entries: 0, generated resources: 0
Workspace has 2 modules
Updating in-memory state...
Sync finished

Timing summary:
BlazeInvocation: 4.6s, Prefetching: 11ms, Other: 380ms
```

Same setup is working in another machine with ubuntu os but not working in MAC with M1 chip.

following is the environment detail. 
Bazel Version : bazel 3.7.2 ;
NDK Version : android-ndk-r18b ;
Android Studio : 4.1.1 ; 

 Any Idea to fix this issue?

**Comment by jiuqiant** [comment]
MediaPipe with its TensorFlow dependence needs Bazel to build everything from scratch. I don't think bazel already supports M1 yet: https://github.com/bazelbuild/bazel/issues/11628.

**Comment by afsaredrisy** [comment]
I fixed it by changing bazel from bazel-3.7-Homebrew to arch -x86_64 brew bazel-3.7
